Added theme for maps

erika-tyagi opened the pull request:
Added a `theme_map_behindbars` (similar to [`theme_void`](https://ggplot2.tidyverse.org/reference/ggtheme.html) but keeps the stuff from `theme_behindbars` that we want to keep). Example of theme output below. 

![maps-ex](https://user-images.githubusercontent.com/47676353/103481481-eced1a80-4da0-11eb-9e36-b67dcf22582f.png)

nmmarquez commented on R/theme_behindbars.R:
```diff
@@ -5,9 +5,9 @@
 #' @param base_family base font family
-#' 
+#'
 #' @examples
```
Can we add an example of how to use the map here for the docs

erika-tyagi replied:
Yes, definitely! 

Separate but related: If we (as a team) like the U.S. state hex maps, I can also add a function to barstools that automates making them per our style guide.

nmmarquez replied:
Love the idea. We should talk about it at our meeting today as well

nmmarquez commented:
Small change but what do yall think about explicitly loading `ggplot2` in addition to `dplyr`? Now that we have these cool themes and map styles I think it makes sense to have it loaded.

erika-tyagi commented:
The facility crosswalks disappeared, so the build will fail, but I think this should be good to go! 
* Added an example for `theme_map_behindbars` 
* Updated the description (depends on `ggplot2` and suggests `usmap` for the example)
